export_erc20_transfers.py

```diff
@@ -45,11 +47,17 @@
 
 args = parser.parse_args()
 
+tlp = None
+if re.compile('^https?://').match(args.provider_uri) == None:
```

Comment:
Not sure if I missed something but HTTPProvider is already handled in `get_provider_from_uri`.

Reply:
ah, conceivable that it was failing due to the blocks I checked had no tx inside.

Reply:
I see :)